src/index.ts

```diff
@@ -224,6 +224,28 @@ function b32decodeXemAddr(data: string): Buffer {
   return nemSdk.default.utils.convert.ua2hex(nemSdk.default.model.address.b32decode(address));
 }
 
+function ontHexToBase58(hexEncoded: Buffer): string {
+  let addr: Buffer;
+  const ADDR_VERSION = '17';
+  if(hexEncoded.toString('hex').length !== 40) {
```

Comment:
Please don't encode to hex just to check length - buffers have lengths too.

Reply:
Done